tutorials/plot_eeg_no_mri.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+"""
+EEG forward operator with a template MRI
+========================================
+
+This tutorial explains how to compute the forward operator from EEG data
+using template MRI subject. We use here the fsaverage brain
+provided by freesurfer.
+
+.. important:: Source reconstruction without an individual T1 MRI from the
+               subject will be less accurate. Do not over interpret
+               activity locations which can be off by multiple centimeters.
+
+.. contents:: This tutorial covers:
+   :local:
+   :depth: 2
+
+"""  # noqa: E501
+
+# Authors: Alexandre Gramfort <alexandre.gramfort@inria.fr>
+#          Joan Massich <mailsik@gmail.com>
+#
+# License: BSD Style.
+
+import os.path as op
+import mne
+from mne.datasets.sample import data_path
```

Comment:
We should use sample here

Reply:
I'll change it

Reply:
oups my bad I typed to quickly and forgot the **not** ... oups...